lib/storage-client.js

```diff
@@ -296,11 +296,10 @@ class ChromedriverStorageClient {
       }
     }
 
-    if (_.isString(minBrowserVersion) && !Number.isNaN(minBrowserVersion)) {
+    if (minBrowserVersion && !Number.isNaN(minBrowserVersion)) {
       // Only select drivers that support the current browser whose major version number equals to `minBrowserVersion`
-      const minBrowserVersionInt = parseInt(minBrowserVersion, 10);
```

Comment:
I think we could keep this cast just to be on the safe side

Reply:
Sure, then let me revert the type to `string | number`